Pat.Subscriber/Pat.Subscriber.csproj

```diff
@@ -22,6 +22,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
-    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="3.1.0" />
+    <PackageReference Include="Microsoft.Azure.ServiceBus" Version="4.1.1" />
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.3.1" />
```

Comment:
Is this second package needed? After a chat with Stijn he suggested only the other package needed updating to support managed identities?

Reply:
Also suggested we just update to `3.2.0` for now